assets/prometheus-operator-user-workload/cluster-role.yaml

```diff
@@ -87,3 +87,15 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Do we know why this was added for prometheus-operator in kube-prom, can't seem to find it in the prometheus-operator repo itself? https://github.com/coreos/prometheus-operator/blob/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml should this be updated there?

Reply:
Without those kube-rbac-proxy didn't work and metrics could be scraped.

Reply:
okay that explains that, thanks!